test/unit/specs/components/DecisioningEngine/decisioningContext.referringPage.spec.js

```diff
@@ -0,0 +1,373 @@
+  return {
+    getPayloadsByType: () => [
+      payloadWithCondition({
+        definition: {
+          conditions: [condition],
+          logic: "and"
+        },
+        type: "group"
+      })
+    ]
+  };
+};
+
+describe("DecisioningEngine:globalContext:referringPage", () => {
+  let storage;
+  let eventRegistry;
+  let decisionProvider;
+  let onResponseHandler;
+  let applyResponse;
+
+  const mockEvent = { getContent: () => ({}), getViewName: () => undefined };
+
+  const setupResponseHandler = (window, condition) => {
+    const contextProvider = createContextProvider({
+      eventRegistry,
+      window
+    });
+
+    onResponseHandler = createOnResponseHandler({
+      decisionProvider,
+      applyResponse,
+      event: mockEvent,
+      decisionContext: contextProvider.getContext()
+    });
+
+    onResponseHandler({
+      response: mockRulesetResponseWithCondition(condition)
+    });
+  };
+
+  beforeEach(() => {
+    storage = jasmine.createSpyObj("storage", ["getItem", "setItem", "clear"]);
+    eventRegistry = createEventRegistry({ storage });
+    decisionProvider = createDecisionProvider();
+    applyResponse = jasmine.createSpy();
+  });
+
+  it("satisfies rule based on matched domain", () => {
+    setupResponseHandler(mockWindow({}), {
+      definition: {
+        key: "referringPage.domain",
+        matcher: "eq",
+        values: ["business.adobe.com", "go.adobe.com"]
+      },
+      type: "matcher"
+    });
+
+    expect(applyResponse).toHaveBeenCalledOnceWith(
+      jasmine.objectContaining({
+        propositions: [proposition]
+      })
+    );
+  });
+
+  it("does not satisfy rule due to unmatched domain", () => {
+    setupResponseHandler(
+      mockWindow({
+        referrer:
+          "https://pro.mywebsite.org:8080/about?m=1&t=5&name=richard#home"
+      }),
+      {
+        definition: {
+          key: "referringPage.domain",
+          matcher: "co",
+          values: ["www.google.com", "www.bing.com"]
+        },
+        type: "matcher"
+      }
+    );
+
+    expect(applyResponse).toHaveBeenCalledOnceWith(
+      jasmine.objectContaining({
+        propositions: []
+      })
+    );
+  });
+
+  it("satisfies rule based on matched referringPage subdomain", () => {
+    setupResponseHandler(mockWindow({}), {
+      definition: {
+        key: "referringPage.subdomain",
+        matcher: "co",
+        values: ["business", "bing"]
+      },
+      type: "matcher"
+    });
+
+    expect(applyResponse).toHaveBeenCalledOnceWith(
+      jasmine.objectContaining({
+        propositions: [proposition]
+      })
+    );
+  });
+
+  it("does not satisfy rule due to unmatched subdomain", () => {
+    setupResponseHandler(
+      mockWindow({
+        referrer:
+          "https://pro.mywebsite.org:8080/about?m=1&t=5&name=richard#home"
+      }),
+      {
+        definition: {
+          key: "referringPage.subdomain",
+          matcher: "eq",
+          values: ["business", "bing"]
```

Comment:
remove "business"

I'm not going to call this out everywhere, but please go through each test and narrow the scope of the test so it is as specific as possible.  remove any extra values that aren't important to what you are testing.  Let's stick to just a single value to test in these definitions unless there is a good reason not to.